testsuite/src/main/java/io/netty/testsuite/transport/socket/DatagramUnicastTest.java

```diff
@@ -161,7 +284,7 @@ public void channelRead0(ChannelHandlerContext ctx, Object msgs) throws Exceptio
 
         Channel sc = null;
         BindException bindFailureCause = null;
-        for (int i = 0; i < 3; i ++) {
+        for (int i = 0; i < 3; i++) {
```

Comment:
seems like the reason for this loop is for unknown failures [1]. I wonder if we should investigate this as a followup issue?

[1] https://github.com/netty/netty/commit/4d79be44efd9de6e582712033fc1f12d6bf2862b